recipes/graphics/xorg/xserver.yaml

```diff
@@ -0,0 +1,105 @@
+inherit: [meson]
+
+metaEnvironment:
+    PKG_VERSION: "21.1.18"
+    PKG_DESCRIPTION: "X.Org X Server"
+    PKG_LICENSE_FILE: "COPYING"
+
+checkoutSCM:
+    scm: url
+    url: https://xorg.freedesktop.org/releases/individual/xserver/xorg-server-${PKG_VERSION}.tar.xz
+    digestSHA256: "c878d1930d87725d4a5bf498c24f4be8130d5b2646a9fd0f2994deff90116352"
+    stripComponents: 1
+
+depends:
+    - graphics::xorg::proto::xorgproto
+    - graphics::xorg::utils::util-macros
+
+    - graphics::xorg::xkbcomp-dev
+    - graphics::fonts::font-util-dev
+    - libs::libepoxy-dev
+
+    - libs::xcb::util-image-dev
+    - libs::xcb::util-keysyms-dev
+    - libs::xcb::util-renderutil-dev
+    - libs::xcb::util-wm-dev
+
+    - libs::xorg::libfontenc-dev
+    - libs::xorg::libpciaccess-dev
+    - libs::xorg::libxkbfile-dev
+    - libs::xorg::libX11-dev
+    - libs::xorg::libXau-dev
+    - libs::xorg::libXaw-dev
+    - libs::xorg::libxcvt-dev
+    - libs::xorg::libXdamage-dev
+    - libs::xorg::libXdmcp-dev
+    - libs::xorg::libXext-dev
+    - libs::xorg::libXfixes-dev
+    - libs::xorg::libXfont2-dev
+    - libs::xorg::libXi-dev
+    - libs::xorg::libXinerama-dev
+    - libs::xorg::libXmu-dev
+    - libs::xorg::libXpm-dev
+    - libs::xorg::libXrender-dev
+    - libs::xorg::libXshmfence-dev
+    - libs::xorg::libXt-dev
+    - libs::xorg::libXv-dev
+    - libs::xorg::libXxf86vm-dev
+    - libs::pixman-dev
+    - libs::mesa3d-dev
+    - libs::libdrm-dev
```

Comment:
I guess this should probably be some virtual packages. I'm lost with all the different gl/gles/egl libraries. Maybe @sixtyfourktec could shed some light on it.

BTW, I noticed that recipes/virtual/libs/libegl.yaml directly forwards to libs::mesa3d. I guess this will blow up sooner than later because we have multiple virtual packages that point to the same real package. IMHO the mesa3d package needs to be split into the various libraries...

Reply:
I changed this to use the virtual package. Not sure if it works with other `libdrm`s :shrug: 

> BTW, I noticed that recipes/virtual/libs/libegl.yaml directly forwards to libs::mesa3d. I guess this will blow up sooner than later because we have multiple virtual packages that point to the same real package. IMHO the mesa3d package needs to be split into the various libraries...

I think this PR is large enough? I'd suggest to create a issue for this to fix it $later...